index.html

```diff
@@ -11,9 +11,11 @@
             display: flex;
             justify-content: center;
             align-items: center;
-            font-family: Helvetica, Arial, sans-serif;;
+            font-family: 'Paytone One', Helvetica, Arial, sans-serif;
             background: #ff6b00;
-            color: #000;
+            color: #fff;
```

Comment:
This colour doesn't have good enough constrast ratio with background.

Reply:
![nfq](https://www.nfq.lt/public/files/logotype/nfq-logo.png)